pyrdp/player/MainWindow.py

```diff
@@ -3,10 +3,12 @@
 # Copyright (C) 2018, 2019 GoSecure Inc.
```

Comment:
```suggestion
# Copyright (C) 2018-2020 GoSecure Inc.
```

You can call me the copyright bumper artist. :sunglasses: ... I know, it's pathetic. :sweat_smile: